Assets/Scenes/PerformanceStressTestSample/PerformanceStressTest.unity

```diff
@@ -2644,7 +2643,7 @@ MonoBehaviour:
   m_GameObject: {fileID: 803874765}
   m_Enabled: 1
   m_EditorHideFlags: 0
-  m_Script: {fileID: 11500000, guid: f492d2ccc97634d44ba5b241f141deb2, type: 3}
+  m_Script: {fileID: 11500000, guid: 9f7f072c827ea4546989fb95f9548627, type: 3}
```

Comment:
The script components of the prefabs used in Performance scene also needed to reset.
it might be easier to revert the guid change caused by the folder rename.

Reply:
Are you sure I didn't get them all? I'm not seeing any warnings or errors about missing scripts when I run the performance scene. I would have expected to see some.

Reply:
You're right that the correct way to do it would be to revert them - I got a little lazy though. If there still are some missing I'll go back and fix them the right way.

Reply:
The prefabs were disabled until starting the test. That might be the reason.